schema.sql

```diff
@@ -0,0 +1,218 @@
+--------------------------------------------------------
+--  File created - Tuesday-May-24-2016   
+--------------------------------------------------------
+--------------------------------------------------------
+--  DDL for Table MUS_ALBUM
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_ALBUM" 
+   (	"ID" NUMBER, 
+	"ARTIST_ID" NUMBER, 
+	"TITLE" VARCHAR2(999), 
+	"EDITION" VARCHAR2(999), 
+	"RELEASED" DATE, 
+	"CREATED" DATE, 
+	"STATUS" NUMBER
+   ) ;
+--------------------------------------------------------
+--  DDL for Table MUS_ARTIST
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_ARTIST" 
+   (	"ID" NUMBER, 
+	"NAME" VARCHAR2(999), 
+	"CREATED" DATE
+   ) ;
+--------------------------------------------------------
+--  DDL for Table MUS_SHELF
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_SHELF" 
+   (	"ID" NUMBER, 
+	"NAME" VARCHAR2(100), 
+	"CREATED" DATE
+   ) ;
+--------------------------------------------------------
+--  DDL for Table MUS_SHELF_ALBUM_MAP
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_SHELF_ALBUM_MAP" 
+   (	"SHELF_ID" NUMBER, 
+	"ALBUM_ID" NUMBER, 
+	"CREATED" DATE
+   ) ;
+--------------------------------------------------------
+--  DDL for Table MUS_STATUS
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_STATUS" 
+   (	"ID" NUMBER, 
+	"NAME" VARCHAR2(100)
+   ) ;
+REM INSERTING into MUS_ALBUM
+SET DEFINE OFF;
```

Comment:
Let's separate the test data from the DDL, constraints and trigger. Just create a separate file where you keep the test data.